app/src/main/java/app/olauncher/ui/HomeFragment.kt

```diff
@@ -248,7 +248,7 @@ class HomeFragment : Fragment(), View.OnClickListener, View.OnLongClickListener
 
     private fun launchApp(appName: String, packageName: String, userString: String) {
         viewModel.selectedApp(
-            AppModel(appName, packageName, getUserHandleFromString(requireContext(), userString)),
+            AppModel(appName, null, packageName, getUserHandleFromString(requireContext(), userString)),
```

Comment:
packageName is null here. Wouldn't that cause a problem with launching apps?

Reply:
The second parameter has been replaced by an optional Collationkey, the package name is the third parameter now.